### PR TITLE
All Endpoints Done

### PR DESCRIPTION
**Pull request by chamodshehanka**
@sajeetharan Can you provide a review

### CODE REVIEW

**Review by Safnaj — COMMENTED** [review]
Perfect..!

**Comment by chamodshehanka** [comment]
> Perfect..!

@Safnaj u haven't approve it

**Review by Safnaj — DISMISSED** [review]

**Comment by Safnaj** [comment]
> > Perfect..!
> 
> @Safnaj u haven't approve it

Wait till @sajeetharan review it.

File: src/api/admin/admin.controller.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import { Request, Response } from "express";
+import * as mongodb from "mongodb";
+import { MongoHelper } from "../../config/mongodb.config";
+import Admin from "./admin.class";
+import ErrorCodes from "../../config/error.codes";
+import SuccessCodes from "../../config/success.codes";
+import * as responses from "../../helpers/responses.handler";
+
+const getCollection = () => {
+  return MongoHelper.client.db("devhitch").collection("events");
```
Comment (sajeetharan) [inline src/api/admin/admin.controller.ts]: Kindly move this to config!

File: src/api/admin/admin.controller.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+export default class AdminController {
+  /**
+   * Add Event
+   * @returns success or failure
+   * @param req
+   * @param res
+   */
+  public addAdmin = async (req: Request, res: Response): Promise<any> => {
+    const requestData = req.body;
+    const collection: any = getCollection();
```
Comment (sajeetharan) [inline src/api/admin/admin.controller.ts]: What if getCollection return an error? how is it handled?

File: src/api/admin/admin.controller.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  };
+
+  /**
+   * Delete Admin
+   * @param req
+   * @param res
+   */
+  public deleteAdmin = async (req: Request, res: Response): Promise<any> => {
+    const id = req.params.id;
+    const collection: any = getCollection();
```
Comment (sajeetharan) [inline src/api/admin/admin.controller.ts]: Handle the error as mentioned above!

File: src/api/index.ts [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+import group from "./group/group.route";
 
 const router: Router = Router();
 
 router.get("/", (_req, res) => {
   res.send("Response from DevHitch Server");
 });
 
+router.use("/admin", admin);
 router.use("/event", events);
```
Comment (sajeetharan) [inline src/api/index.ts]: use just event , similar to other routes

File: src/config/error.codes.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export default {
+  UNAUTHORIZED: "UNAUTHORIZED",
+  REGISTRATION_FAILED: "REGISTRATION_FAILED",
```
Comment (sajeetharan) [inline src/config/error.codes.ts]: Remove the _ in the values

File: src/api/admin/admin.controller.ts (hunk 1 above)
Comment (chamodshehanka) [reply src/api/admin/admin.controller.ts]: Done ✅

File: src/config/error.codes.ts (hunk 5 above)
Comment (chamodshehanka) [reply src/config/error.codes.ts]: Done ✔️

File: src/api/index.ts (hunk 4 above)
Comment (chamodshehanka) [reply src/api/index.ts]: Done

**Pull request merged**